api/workAllocation2/roleService.ts

```diff
@@ -12,3 +13,15 @@ export async function handleGetRolesByCaseId(path: string, req: EnhancedRequest)
   const headers = setHeaders(req);
   return await httpMock.get<CaseRole>(path, {headers});
 }
+
+export function handleShowAllocatorLinkByCaseId(caseId: string, req: EnhancedRequest): boolean {
```

Comment:
@kuda1992  I can see you looking for caseId below. But, you also need to look if the the user has the case allocator role. That is missing. Could you please add it? Thanks